flink-ml-lib/src/main/java/org/apache/flink/ml/feature/vectorassembler/VectorAssemblerParams.java

```diff
@@ -21,11 +21,31 @@
 import org.apache.flink.ml.common.param.HasHandleInvalid;
 import org.apache.flink.ml.common.param.HasInputCols;
 import org.apache.flink.ml.common.param.HasOutputCol;
+import org.apache.flink.ml.param.IntArrayParam;
+import org.apache.flink.ml.param.Param;
+import org.apache.flink.ml.param.ParamValidators;
+
+import java.util.Arrays;
 
 /**
```

Comment:
Would it be better to add some document describing how users need to tackle the sizes parameter? For example, Spark has some descriptions of the function of VectorSizeHint in the JavaDoc of `VectorAssembler.handleInvalid`. Maybe we can add similar descriptions to the JavaDoc of `VectorAssembler`.